index.d.ts

```diff
@@ -135,6 +135,17 @@ declare namespace execa {
 		*/
 		readonly stdio?: 'pipe' | 'ignore' | 'inherit' | readonly StdioOption[];
 
+		/**
+		When using the `stdio: 'ipc'` option or `execa.node()`, how messages passed to [`childProcess.send()`](https://nodejs.org/api/child_process.html#child_process_subprocess_send_message_sendhandle_options_callback) are serialized:
```

Comment:
```suggestion
		When using the `stdio: 'ipc'` option or `execa.node()`, this specifies how messages passed to [`childProcess.send()`](https://nodejs.org/api/child_process.html#child_process_subprocess_send_message_sendhandle_options_callback) are serialized:
```

Reply:
I think the Node.js description is a bit clearer though:

> Specify the kind of serialization used for sending messages between processes.

Maybe:

> Specify the kind of serialization used for sending messages between processes when using `execa.node()` or the `stdio: 'ipc'` option.

?

Reply:
We should also include the required Node.js version.

Reply:
Fixed in 6c7f4938d5712e39d89a2049219ee51821993517